src/components/UncontrolledTabs.js

```diff
@@ -22,18 +22,21 @@ function isTabDisabled(node) {
 }
 
 let canUseActiveElement;
-try {
-  canUseActiveElement = !!(
-    typeof window !== 'undefined' &&
-    window.document &&
-    window.document.activeElement
-  );
-} catch (e) {
-  // Work around for IE bug when accessing document.activeElement in an iframe
-  // Refer to the following resources:
-  // http://stackoverflow.com/a/10982960/369687
-  // https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/12733599
-  canUseActiveElement = false;
+
+function determineCanUseActiveElement(environment) {
```

Comment:
Wrapping this into a function that runs only once (since initially `canUseActiveElement` is `undefined`) to be able to use the `environment` from the props instead of assuming `window`.